rwengine/src/audio/SoundSource.cpp

```diff
@@ -148,7 +148,7 @@ bool SoundSource::findAudioStreamSfx() {
     }
 
     audioStream = formatContext->streams[streamIndex];
-    codec = avcodec_find_decoder(audioStream->codecpar->codec_id);
+    auto codec = avcodec_find_decoder(audioStream->codecpar->codec_id);
```

Comment:
From here the `auto` should also be removed because it shadows the member variable:

```suggestion
    codec = avcodec_find_decoder(audioStream->codecpar->codec_id);
```